src/metabase/core.clj

```diff
@@ -122,6 +123,9 @@
       ;; otherwise update if appropriate
       (sample-data/update-sample-database-if-needed!)))
 
+  (when (slack/slack-configured?)
+    (slack/refresh-cache!))
+
```

Comment:
I wasn't sure if this was appropriate, all it does is (potentially) fire off some http requests, and set the slack-cache setting.

Reply:
Won't this block until the Slack requests return? If so we probably shouldn't do it during startup.

Reply:
Good catch, I'll wrap this in a future, I didn't want to block here either